src/Files.Shared/Enums/GroupByDateUnit.cs

```diff
@@ -0,0 +1,11 @@
+﻿// Copyright (c) 2023 Files Community
+// Licensed under the MIT License. See the LICENSE.
+
+namespace Files.Shared.Enums
+{
+	public enum GroupByDateUnit : byte
+	{
+		Year = 0,
+		Month = 1
```

Comment:
When creating enums, always please add comments.

```suggestion
		/// <summary>
		/// Group items by year.
		/// </summary>
		Year = 0,
		
		/// <summary>
		/// Group items by month.
		/// </summary>
		Month = 1
```